### DISCUSSION TITLE
[Feature]: Private photos which do not show up in timeline

### DISCUSSION THREAD

**Discussion by colin715** [immich-app/immich#1638] [Feature Request]
### Feature detail

I would like the ability to mark photos as "Private", which would cause them to be hidden from the main timeline view.  You could then have a "Private" section on the sidebar which would house all of these.  Ideally, you would still be able to add private photos to an album and would be able to see the photos if you click on the album.  If all photos in an album are private, there should not be a thumbnail for that album.

### Platform

Mobile App

**Comment by jcastro** [comment]
Would be great to be able to create an album as well that doesn’t show up in the timeline

**Reply by colin715** [reply]
I really like this suggestion, as I would put all of my "private" photos into a specific album anyways so it would eliminate a step.

**Reply by Mougi** [reply]
I would love to see this as a feature in the future!

**Comment by NickSkier** [comment]
It's almost like archive feature in Google photos. So, yeah, it would be nice to see something like that

**Comment by ishioni** [comment]
Additionally it could maybe be password protected or something like that?

**Comment by correctomundo79** [comment]
https://github.com/immich-app/immich/discussions/1625
Is a similar feature request.

**Comment by colin715** [comment]
PR for archiving individual assets - https://github.com/immich-app/immich/pull/2225

**Comment by jrasm91** [comment]
Archiving has been implemented on web and mobile in v1.54.

**Reply by pauloantoniassi** [reply]
IMHO, archiving is not the same as private. Archiving is for stuff that I don't use, but don't want to delete. Private is for stuff that I can not risk showing someone by accident (like "NSFW photos" or valuable data). 
Sure, the feature is the same, but 2 different objectives. For the private section on mobile, I would even add a fingerprint unlock, while the archive is just random stuff.

**Reply by alceasan** [reply]
I'd like also to open again this feature requests, as others have said it's not the same to have archived photos (that at some point could be deleted). "Private" photos are valuable, but you don't want anyone else to see them (for whatever reason) or visible in the timeline.

Photoprism has this funcionality separated from the archived photos: [https://docs.photoprism.app/user-guide/organize/private/](https://docs.photoprism.app/user-guide/organize/private/)

**Reply by jrasm91** [reply]
You can track/vote for private photos here: https://github.com/immich-app/immich/discussions/2451
